docs/src/toml-files.md

```diff
@@ -22,11 +22,20 @@ are described below.
 
 ### The `authors` field
 
-For a package, the optional `authors` field is a list of strings describing the
-package authors, in the form `NAME <EMAIL>`. For example:
+For a package, the optional `authors` field is a TOML array describing the package authors. 
+Entries in the array can either be a string in the form `NAME <EMAIL>`, or arrays with the following keys:
+ - `name` (required): the name of the author
+ - `email`: email address
+ - `orcid`: 16-digit [ORCID](https://orcid.org/) identifier as a string
+ - `github`: [GitHub](https://github.com/) username
```

Comment:
Baking GitHub in here feels weird. How about link and it can be any site. If it happens to be a github  user url then you can infer that its their github handle.

Reply:
That's a fair point, which I somewhat agree with. The main reason to include it is that GitHub is the primary platform/network most Julia package developers use.

The downside of a generic URL is that there can then only be 1: I guess we can have "urls" be a list?

Reply:
Don't you think that each person can pick one link that's identifying for them?

Reply:
True.